autopts/bot/common_features/report.py

```diff
@@ -68,106 +69,176 @@ def make_repo_status(repos_info):
 # ****************************************************************************
 # .xlsx spreadsheet file
 # ****************************************************************************
-def make_report_xlsx(report_xlsx_path, results_dict, status_dict, regressions_list,
-                     progresses_list, descriptions, xmls, errata):
-    """Creates excel file containing test cases results and summary pie chart
+def parse_result_column(full_result: str) -> Tuple[str, str]:
+    """
+    Helper function: Searches for fixed keywords in `full_result`.
+    If found, returns (keyword, rest_text).
+    If it doesn't find it, it returns (full_result, "").
+    """
+    # The order matters - if there is a possibility
+    # of several matches, the first one in the list decides.
+    patterns = [
```

Comment:
There are more possible results. If I remember correctly, only UNKNOWN VERDICT will contain appended additional info, like:
`UNKNOWN VERDICT: Timeout. Adjust TSPX_time_guard value if user need more time.`
, because of this line:
https://github.com/auto-pts/auto-pts/blob/65e598fcf8d83b720406a6c02651e06c74ab3bc0/autopts/ptscontrol.py#L175

`UNKNOWN VERDICT: None`

```suggestion
    patterns = ["UNKNOWN VERDICT"]
```

We could refactor this later in other PR, to skip this parsing and keep additional info in dedicated variable.